crates/msr/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "ccp-msr"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cpu-utils = {workspace = true}
+thiserror = { workspace = true }
+nix = { version = "0.27.1" , features = ["uio"] }
```

Comment:
Let's specify versions in the workspace root and then just use { workspace = true} here.